aireview/poetry.lock

```diff
@@ -1,5 +1,188 @@
 # This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
 
```

Comment:
**style**: Review the locked dependency versions for potential security vulnerabilities. Consider adding a CI step to scan dependencies using tools like safety or snyk.